FCMExternalProberTarget/app/src/main/java/com/google/fcmexternalprobertarget/FCMReceiveService.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
This source header is good, but we need a lot more of them.  Like an annoying amount of source headers.

"BEST PRACTICE: Add license headers to any files that can be added to (i.e. anything that takes the format of a source file and supports file comments). This would include things like YAML files."

https://opensource.google/docs/releasing/preparing/#license-headers